docs/extensions/napoleon_django/docstring.py

```diff
@@ -29,6 +29,12 @@
                  can_collapse=True),
 ])
 
+# Similar to the extensions above, but this rewrites the 'variable' type used for class attrs to
+# use the data rolename, which prevents sphinx from attempting to cross-reference class attrs.
```

Comment:
Can you include why it is important to prevent sphinx from attempting to cross-reference class atts?